config/routes.rb

```diff
@@ -110,6 +110,7 @@
   resources :notifications, only: :index
   resources :other_duties, only: %i[new create edit index update]
   resources :missing_data_reports, only: %i[index]
+  resources :learning_hours, only: %i[new create edit update]
```

Comment:
The learning hours are already defined on line 146 in the routes file, so this should not be necessary.

Reply:
Oh yeah, I see. There is a nested learning_hours resource already declared. This makes this task a little bit tricky but I think I'll come up with a solution.